docs/installation.rst

```diff
@@ -4,105 +4,226 @@
-~~~~~~~
+   .. group-tab:: Windows
 
-1. `Download Node.js <https://nodejs.org/en/download/>`__ from the official website and install Node.js using the downloaded installer.
-2. Make sure both ``node --version`` and ``npm --version`` work in your Command Prompt.
+      - `Download Node.js <https://nodejs.org/en/download/>`__ from the official website and install it using the downloaded installer
+      - Make sure both ``node --version`` and ``npm --version`` work in your Command Prompt
+      - Node.js needs to be at least version 6
 
-Linux
-~~~~~
+.. note::
+   You can also use `nvm <https://github.com/creationix/nvm>`__ to install Node.js on your computer.
 
-1. `Install Node.js as system package <https://nodejs.org/en/download/package-manager/>`__.
-2. Make sure both ``node --version`` and ``npm --version`` work in your Terminal.
+.. warning::
+   If your internet connection is restricted (VPN, firewall, proxy), you need to run following additional commands to configure npm:
 
-Pro Tips
-~~~~~~~~
+   .. code-block:: text
 
--  :ref:`Continuous Integration section in the How-To Guides <continuous-integration>` can help you to install Dredd on CI server.
--  To maintain multiple Node.js versions on your computer, check out `nvm <https://github.com/creationix/nvm>`__.
+      npm config set proxy "http://proxy.example.com:8080"
+      npm config set https-proxy "https://proxy.example.com:8080"
 
-Install Dredd
--------------
+   Otherwise you'll get similar errors during Dredd installation:
 
-1. ``npm install -g dredd``
-2. ``dredd --version``
+   .. code-block:: text
 
-If the second command works, you’re done!
+      npmERR! Cannot read property 'path' of null
+      npmERR!code ECONNRESET
+      npmERR!network socket hang up
 
-Globally vs locally
-~~~~~~~~~~~~~~~~~~~
+   Later be sure to read :ref:`how to set up Dredd to correctly work with proxies <using-https-proxy>`.
 
-The ``-g`` ensures Dredd will be installed “globally”. That means you’ll be able to access it from any directory just by typing ``dredd``.
 
-If you work on projects installable by ``npm``, i.e. projects containing ``package.json``, you might want to have Dredd installed as a development dependency instead. Just install Dredd by ``npm install dredd --save-dev``. See ``package.json`` of the `Dredd Example <https://github.com/apiaryio/dredd-example/>`__ repository for inspiration.
+.. _install-git:
 
-Which Version?
+Installing Git
 ~~~~~~~~~~~~~~
 
--  **For development**, always go with the latest version.
--  **For testing in CI** (:ref:`what’s CI? <continuous-integration>`), always pin your Dredd version to a specific number and upgrade to newer releases manually (but often!).
+Git is needed to install some of Dredd's dependencies (:ghissue:`gavel.js#83`).
+
+.. tabs::
+
+   .. group-tab:: macOS
+
+      - First try whether Git isn't already available by running ``git --version``
+      - If not and you're using `Homebrew <https://brew.sh/>`__, run ``brew install git``
```

Comment:
According to this https://git-scm.com/book/en/v2/Getting-Started-Installing-Git it should be available by default.